src/Tempest/Database/src/Connections/PostgresConnection.php

```diff
@@ -52,4 +55,9 @@ public function dialect(): DatabaseDialect
     {
         return DatabaseDialect::POSTGRESQL;
     }
+
+    public function tableNamingStrategy(): NamingStrategy
```

Comment:
We usually rely on public properties directly instead of getters. We plan on removing most getters from Tempest when PHP 8.4 arrives with property hooks